docs/package.json

```
@@ -26,7 +26,7 @@
     "@docusaurus/core": "^2.0.0-alpha.61",
     "@docusaurus/preset-classic": "^2.0.0-alpha.60",
     "@mdx-js/react": "^1.5.8",
-    "@stellar/lyra-api": "1.0.0-alpha.2",
+    "@stellar/lyra-api": "1.0.0",
```

ah okay - so alpha.3 is the latest. I forgot to update this when I pushed out the new version :(
let's be specific about which version we're using to help us debug in the future. This change should fix lyra-docs.netlify.app

```suggestion
    "@stellar/lyra-api": "1.0.0-alpha.3",
```